packages/calcite-components/src/components/handle/handle.e2e.ts

```diff
@@ -37,6 +37,51 @@ describe("calcite-handle", () => {
     expect(await handle.getProperty("activated")).toBe(true);
   });
 
+  it("sets activated to false when blurred", async () => {
```

Comment:
Does this fix any behavior in list or another consuming component? If so, is it worth adding a check there? Disregard if not the case.